ManatiCR/manatime
# 5: Content type projects.

**Steps to test**
- Run the script manatime_local_install.sh.
- See if the features manatime_feature_projects was added and activated.
- Add a new project node.

**DoD**
- As an administrator I should be able to go to node/add/project and add a new project with the described fields
- As an administrator, I should be able to view a project (no themed) and click on the Edit tab and update that project.

[issue link](https://github.com/ManatiCR/manatime/issues/5)
